src/VscPreviewExtension/Microsoft.DocAsCode.Dfm.VscPreview/PreviewCommand.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Dfm.VscPreview
+{
+    using System.Collections.Generic;
+    using System.IO;
+
+    using Microsoft.DocAsCode.Common;
+
+    public class PreviewCommand
+    {
+        public static PreviewJsonConfig ParsePreviewCommand(string baseDir)
+        {
+            string configFilePath = Path.Combine(baseDir, PreviewConstants.ConfigFile);
+            PreviewJsonConfig config = new PreviewJsonConfig();
+            if (!string.IsNullOrEmpty(PreviewConstants.ConfigFile) && File.Exists(configFilePath))
```

Comment:
>!string.IsNullOrEmpty(PreviewConstants.ConfigFile) [](start = 16, length = 50)

can remove the first check? for one reason it's const, another reason is File.Exists will check?